trial-config-service/README.md

```diff
@@ -0,0 +1,14 @@
+# trial-config-service
+
+The following properties are required:
+
+    -Djdbc.driver=**db driver class name** 
```

Comment:
I recommend to obtain credentials from KeyVault. You can find an example in the `sample-service`